src/main/resources/templates/fragments/inputs/text.html

```diff
@@ -1,31 +0,0 @@
-<th:block
-    th:fragment="text"
-    th:with="
-      hasHelpText=${!#strings.isEmpty(helpText)},
-      hasLabel=${!#strings.isEmpty(label)},
-      hasAriaLabel=${!#strings.isEmpty(ariaLabel)},
-      hasValue=${value != null},
-      hasError=${
-        errorMessages != null &&
-        errorMessages.get(inputName) != null &&
-        (#arrays.length(errorMessages.get(inputName)) > 0) }"
-    th:assert="${!#strings.isEmpty(inputName)}, ${hasLabel || hasAriaLabel}">
-  <div th:class="'form-group' + ${(hasError ? ' form-group--error' : '')}">
-    <label th:if="${hasLabel}" th:for="${inputName}" th:text="${label}" class="form-question"/>
-    <p class="text--help"
-       th:if="${hasHelpText}"
-       th:id="${inputName + '-help-text'}"
-       th:text="${helpText}"></p>
-    <input type="text" class="text-input"
-           th:id="${inputName}"
-           th:name="${inputName}"
-           th:placeholder="${placeholder}"
-           th:attr="
-            aria-describedby=${hasHelpText ? inputName + '-help-text' : ''},
-            aria-labelledby=${hasAriaLabel ? ariaLabel : ''},
-            aria-invalid=${hasError}"
-           th:value="${hasValue ? value : fieldData.getOrDefault(inputName, '')}">
```

Comment:
This spot allows for an override of the data filled into the field. I don't think we need this in MD's code.  But maybe we should pull this logic into the ffb?